### PR TITLE
Changed the budget extension description

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Changed the description of the Budget extension mail.
```
Summary of you budget request TicketId: -

Allocated Budget: $ 24
Cost as of today: $ 94
Exceeded Budget: $ 70
If you want to keep using this allocation, please extend the budget:

Click here to open Budget extend page
Fill the details, and select the TicketId: ``
Enter the amount and description of the why you need extension.
Then click on submit
If you don't need to use this allocation anymore, you can ignore this email. Please don't forgot to terminate the resources,
You won't receive additional emails as long as the allocation cost does not increase.
```

### CODE REVIEW

File: cloud_governance/common/mails/templates/cro_monitor_budget_remain_alert.j2 [hunk 1]
```diff
@@ -2,11 +2,20 @@
...
-    Remaining budget will be $ {{remain_budget}}.</p>
-    <p>Open the budget extension if further needed.</p>
-    <p>This ticket will be closed, if the budget will exceeds 110%.</p>
-    <p>You can extend the budget <a href="{{portal}}/extend/budget">here</a>.</p>
-    <p>Visit the <a href="{self.__portal}/wiki/clouds">wiki page</a> to get more information</p>
+    <p>Summary of you budget request TicketId: {{ ticket_id }}</p>
+    <ul>
+        <li>Allocated Budget: <b>$ {{ budget }}</b></li>
+        <li>Cost as of today: <b>$ {{ used_budget }}</b></li>
+        <li>Remaining Budget: <b>$ {{ remain_budget }}</b></li>
```
Comment (kpouget) [inline cloud_governance/common/mails/templates/cro_monitor_budget_remain_alert.j2:9]: can you remove the space after the `$` sign? (for proper formating of the currency)
```suggestion
        <li>Allocated Budget: <b>${{ budget }}</b></li>
        <li>Cost as of today: <b>${{ used_budget }}</b></li>
        <li>Remaining Budget: <b>${{ remain_budget }}</b></li>
```

**Review by kpouget — COMMENTED** [review]
thanks @athiruma , looks good to me :)

File: cloud_governance/common/mails/templates/cro_monitor_budget_remain_alert.j2 [hunk 2]
```diff
@@ -2,11 +2,20 @@
...
 <div>
-    <p>You completed $ {{used_budget}} / $ {{budget}} of budget till today of the TicketId: {{ticket_id}}.
-    Remaining budget will be $ {{remain_budget}}.</p>
-    <p>Open the budget extension if further needed.</p>
-    <p>This ticket will be closed, if the budget will exceeds 110%.</p>
-    <p>You can extend the budget <a href="{{portal}}/extend/budget">here</a>.</p>
-    <p>Visit the <a href="{self.__portal}/wiki/clouds">wiki page</a> to get more information</p>
+    <p>Summary of you budget request TicketId: {{ ticket_id }}</p>
+    <ul>
+        <li>Allocated Budget: <b>$ {{ budget }}</b></li>
```
Comment (ebattat) [inline cloud_governance/common/mails/templates/cro_monitor_budget_remain_alert.j2:7]: Please use the following terms:
Allocated Budget => Budget
Cost as of today => Spending 
Remaining Budget => Remaining

**Review by ebattat — APPROVED** [review]
/LGTM

**Comment by kpouget** [comment]
@athiruma @ebattat, this PR introduced an inconsistency in the email:
```
    Budget: $500
    Spending: $675
    Remaining: $175
```
the last line should read something like `Over-spending: $175`

**Comment by athiruma** [comment]
> @athiruma @ebattat, this PR introduced an inconsistency in the email:
> 
> ```
>     Budget: $500
>     Spending: $675
>     Remaining: $175
> ```
> 
> the last line should read something like `Over-spending: $175`

Created a new [PR](https://github.com/redhat-performance/cloud-governance/pull/661/files) regarding this bug

**Pull request merged**